x/evm/keeper/keeper.go

```diff
@@ -20,36 +20,37 @@ import (
 // Keeper wraps the CommitStateDB, allowing us to pass in SDK context while adhering
 // to the StateDB interface
 type Keeper struct {
-	csdb     *types.CommitStateDB
-	cdc      *codec.Codec
-	blockKey sdk.StoreKey
-	txCount  *count
-	bloom    *big.Int
+	cdc           *codec.Codec
+	storeKey      sdk.StoreKey
```

Comment:
Maybe this could be a better name than what I had, `blockKey` and this, to be clear this is just used for storing `block hash -> height` and `height -> bloom filter` mappings needed for the Web3 API.

Optional to change if you'd like

Reply:
@austinabell I see... What do you think about renaming it to `storeKey` and adding a comment with what you said above?

Reply:
Either way, fine with me, just wanted to give context